### ISSUE TITLE
Cannot compile for attiny2313

### ISSUE THREAD

**Issue by nusov** [teknoman117/avr#1]
Hi,
I got 

$ make
avr-gcc -mmcu=attiny2313 -DF_CPU=8000000 -O2 -fpack-struct -fshort-enums -funsigned-bitfields -funsigned-char -Wall -I.  avr-rtos-isr.s -c -o avr-rtos-isr.o
avr-rtos-isr.s: Assembler messages:
avr-rtos-isr.s:58: Error: illegal opcode mul for mcu attiny2313
avr-rtos-isr.s:91: Error: illegal opcode call for mcu attiny2313
avr-rtos-isr.s:96: Error: illegal opcode mul for mcu attiny2313
make: **\* [avr-rtos-isr.o] Error 1

**Comment by teknoman117** [comment]
Yes, the code for the AVR RTOS only has support for the ATmega164(p), ATmega324(p), and ATmega644(p) at the moment.  I've only written the assembly code for those CPUs so far.
